addon/-private/system/class-meta.js

```diff
@@ -0,0 +1,57 @@
+import EmptyObject from 'ember-data/-private/system/empty-object';
+
+export default class ClassMeta {
```

Comment:
Although implicitly tested, I suspect it will pay dividends to throughly unit test this. It will help keep us from trolling ourselves while we iterate quickly.